spec/factories/payments.rb

```diff
@@ -5,8 +5,14 @@
     end
 
     claims do
+      personal_details = {
+        national_insurance_number: generate(:national_insurance_number),
```

Comment:
although we don't technically check it for matching claims, I wonder if we should also set an explicit `teacher_reference_number` here as otherwise the two claims against the payment will have different ones, which feels off.

Reply:
:+1: